index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <div class="app-bar">
+        <div class="upper-bar">
+          <p class="time">9 : 41</p> 
+          <div class="upper-icons">
+            <p class="ntew"> </p>
+            <p class="wifi"></p>
+            <p class="battery"></p>
+          </div>
+        </div>
+        <div class="lower-bar">
+          <p class="logoo"><img src="image/Logo.png" alt="logo icon"></p> 
+          <div class="menu-icons"> <img class="menuimg" src="image/menu.png" alt="menu icon"></div>
+        </div>
+      </div>
+
+      <div class="bgimage">
+        <p class="myname">Somarib Emmanuel</p>
+        <p class="jobtitle">Software Developer</p> 
+          <p class="abilities">
+            I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,  
+            don’t hestiate to contact me.
+          </p>
+        <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+        <p class="frame38"><img src="image/Frame38.png" alt="language proficiency"></p>
+        <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+        <p class="social"><img src="image/Social_media.png" alt="social media links"></p>
+
```

Comment:
- [ ] Kindly make sure you get each icon image and they're all wrapped in a `ul` tag. Each icon should be wrapped in an `li` tag and should have an `a` tag to help people connect with you 🙏